app/src/main/res/layout/fragment_character_fragment_dialog.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="200dp"
```

Comment:
avoid hardcoding measures, move all the `dp, sp or other` values to `dimens.xml`

Reply:
done